testsuite/formatter.py

```diff
@@ -8,10 +8,14 @@ class ChunkedJsonFormatter(PrettyJSONFormatter):
     """
     def __init__(self, a, b):
         super(ChunkedJsonFormatter, self).__init__(a, b)
-        self.snapshot = []
+        self.snapshot = {
```

Comment:
Instead of converting snapshot into a dictionary, why not make `plan` a top level property as well? Then pass them both independently through `on_snapshot` and `emit`. Mostly I'm thinking that `status` already has a meaning in the behave context, and I don't want them to get confused.